### PR TITLE
Variable name consistency and improved error flow in task chain

### PR DESCRIPTION
**Pull request by turkycat**

### CODE REVIEW

**Comment by turkycat** [comment]
@ras0219-msft please check task chain error flow in BluetoothSerial & NetworkSerial (one change)

**Comment by turkycat** [comment]
@ooeygui after this pull, 1.1 is ready for release

File: source/Serial/NetworkSerial.cpp [hunk 1]
```diff
@@ -155,20 +155,20 @@ NetworkSerial::flush(
     create_task( _current_store_operation )
```
Comment (ras0219-msft) [inline source/Serial/NetworkSerial.cpp]: I still don't understand why _current_store_operation is a member variable. Why not use a lambda capture? In this case, you have some curious behavior if the user issues multiple flush() commands quickly.

File: source/Serial/NetworkSerial.cpp [hunk 2]
```diff
@@ -155,20 +155,20 @@ NetworkSerial::flush(
...
         {
-            task_.get();
-
-            //detect disconnection
-            if( _current_store_operation->Status == Windows::Foundation::AsyncStatus::Error )
-            {
-                _connection_ready = false;
-                ConnectionLost( L"A fatal error has occurred in NetworkSerial::flush() and your connection has been lost." );
-            }
+            task_.wait();
```
Comment (ras0219-msft) [inline source/Serial/NetworkSerial.cpp]: Note that you're dropping the bool result on the ground.

File: source/Serial/BluetoothSerial.cpp [hunk 3]
```diff
@@ -198,20 +198,20 @@ BluetoothSerial::flush(
     create_task( _current_store_operation )
         .then( [ this ]( unsigned int value_ )
```
Comment (ras0219-msft) [inline source/Serial/BluetoothSerial.cpp]: value_ is unused.

File: source/Serial/NetworkSerial.cpp (hunk 1 above)
Comment (turkycat) [inline source/Serial/NetworkSerial.cpp]: that is a good point. previously, we used this store operation to tell if any of the previous store operations failed, and I remember we also had issues with items being prematurely destructed if we didn't store references to them. Now that we are using the store operation immediately, a capture is better.

File: source/Serial/BluetoothSerial.cpp (hunk 3 above)
Comment (turkycat) [inline source/Serial/BluetoothSerial.cpp]: Intended. Will mark as UNREFERENCED_PARAMETER per your suggestion. Thanks!

**Pull request merged**